.gitignore

```diff
@@ -0,0 +1,2 @@
+/composer.lock
+/vendor
```

Comment:
Can you please remove this file

Reply:
OK I don't know why, but I will remove this file.